app/views/admin/tickets/show.html.haml

```diff
@@ -33,5 +34,7 @@
             %td
               = buyer.affiliation
             %td
-              = @ticket.paid?(buyer)
```

Comment:
the `paid? `method is only used here, so you can remove it form the ticket model. You will also have to modify the tests.